CMakeLists.txt

```diff
@@ -27,7 +27,8 @@ find_package(Gaudi)
 find_package(k4FWCore) # implicit: Gaudi
 find_package(EDM4HEP) # implicit: Podio
 find_package(DD4hep)
-find_package(FCCDetectors)
+find_package(k4geo)
+#find_package(FCCDetectors)
```

Comment:
Can this line be removed as well?

Reply:
Hi @kjvbrt I have fixed this line in a new branch in which I rebased the code to main - I am going to close this PR and open a new one